scripts/utils.sh

```diff
@@ -1,12 +1,54 @@
 #!/usr/bin/env bash
 
-# prevent execution of this script, only allow execution
+# prevent execution of this script, only allow sourcing
 $(return >/dev/null 2>&1)
 test "$?" -eq "0" || { echo "This script should only be sourced." >&2; exit 1; }
 
 # exit on errors, undefined variables, ensure errors in pipes are not hidden
 set -Eeuo pipefail
 
+function install_package() {
```

Comment:
Do we need to install packages? 
Should we just fail, saying "Sorry you need XYZ to run this script" ?

Reply:
Ok. As we are not running an exhausive checks on all the kernels, I'll remove the installation part